roles/apply_nmstate/tasks/main.yml

```diff
@@ -7,11 +7,11 @@
         name: nmstate
         state: present
 
-    - name: "Copy rendered_nmstate_yml to {{ vm_nmstate_config_path }}"
-      copy:
-        content: "{{ rendered_nmstate_yml }}"
-        dest: "{{ vm_nmstate_config_path }}"
-        mode: 0644
+   # - name: "Copy rendered_nmstate_yml to {{ vm_nmstate_config_path }}"
```

Comment:
I had to comment it out because it was giving me error while appyling the nmstate yml in the kvm host. Error was about dns-resolver thing. You might remember it from previous cases as well couple of months ago. I had to run the nmstate yml manually so I commented out this part in the roles but I can remove the comments from this part.

Reply:
If you don't want crucible to apply the nmstate to the vmhost you can add the flag 
`SETUP_VM_BRIDGE: False` to your inventory. :) Saving you having to comment it out :)

Reply:
Done